src/core/thread/thread_tlvs.hpp

```diff
@@ -484,7 +484,7 @@ class ThreadNetworkDataTlv: public ThreadTlv
      * @retval TRUE  the TLV appears to be well-formed.
      *
      */
-    bool IsValid(void) const { return GetLength() <= sizeof(*this) - sizeof(ThreadTlv); }
+    bool IsValid(void) const { return true; }
```

Comment:
At this point, why should IsValid exist?

Reply:
The TLV parsing/processing code uses `IsValid()` on TLVs.  Keeping `IsValid()` on this TLV keeps the code consistent with the rest of the TLV parsing code.